tests/providers/influxdb/hooks/test_influxDbHook.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
You missed the rename of the file from `test_influxDBHook.py` to `test_influxdb.py`

Reply:
Added unit tests, @eladkal , thanks